com.unity.render-pipelines.core/Runtime/Volume/VolumeComponent.cs

```diff
@@ -195,10 +200,14 @@ public virtual void Override(VolumeComponent state, float interpFactor)
         /// <param name="state">The value to set the state of the overrides to.</param>
         public void SetAllOverridesTo(bool state)
         {
-            SetAllOverridesTo(parameters, state);
+            SetOverridesTo(parameters, state);
         }
 
-        void SetAllOverridesTo(IEnumerable<VolumeParameter> enumerable, bool state)
+        /// <summary>
+        /// Sets the override state of the given parameters on this component to a given value.
+        /// </summary>
+        /// <param name="state">The value to set the state of the overrides to.</param>
+        public void SetOverridesTo(IEnumerable<VolumeParameter> enumerable, bool state)
```

Comment:
Isn't having this public is redudant with SetAllOverridesTo?

If usefull keep this change, if not really remove it. We only expose what is needed as exposed API are hard to change.

Reply:
I need to be public. See `VolumesComponentEditor.SetOverridesTo`